test/integration/local/test_horovod.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2017-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import json
+import os
+import tarfile
+
+import pytest
+from sagemaker.mxnet import MXNet
+
+from integration import RESOURCE_PATH
+
+
+@pytest.mark.skip_gpu
```

Comment:
did you want to add a GPU version in the same vein as https://github.com/aws/sagemaker-tensorflow-training-toolkit/issues/388?

Reply:
Addressed.